jobs/mariadb-blacksmith-plans/templates/plans/standalone/init

```diff
@@ -4,5 +4,6 @@ set -eu
 
 safe gen $CREDENTIALS/standalone/system password
 safe gen $CREDENTIALS/standalone/system username
+safe set $CREDENTIALS/standalone/system dbname="mysql"
```

Comment:
Please change this to `$CREDENTIALS/standalone/database:name`

Reply:
Updated